common/borders/borders-en.html

```diff
@@ -0,0 +1,77 @@
+---
+{
+	"title": "Borders",
+	"language": "en",
+	"altLangPage": "borders-fr.html",
+	"breadcrumbs":
+	[
+		{
+			"title": "GCWeb home",
+			"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+		}
+	],
+	"secondlevel": false,
+	"dateModified": "2023-06-20",
+	"share": "true"
+}
+---
+
+<h3>CSS Class</h3>
+<dl class="dl-horizontal">
+	<dt><code>.brdr-left</code></dt>
+	<dd>Set a border to the left of the element</dd>
+	<dt><code>.brdr-tp</code></dt>
+	<dd>Set a border on top of the element</dd>
+	<dt><code>.brdr-rght</code></dt>
+	<dd>Set a border to the right of the element</dd>
+	<dt><code>.brdr-bttm</code></dt>
+	<dd>Set a border below the element</dd>
+	<dt><code>.brdr-0</code></dt>
+	<dd>Remove borders</dd>
+	<dt><code>.brdr-rds</code></dt>
+	<dd>Remove border radius</dd>
+</dl>
+<div class="row">
+	<div class="col-md-6">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h3 class="mrgn-tp-0">Appearance</h3>
+				<p class="brdr-lft">
+					Example 1
+				</p>
+				<p class="brdr-tp">
+					Example 2
+				</p>
+				<p class="brdr-rght">
+					Example 3
+				</p>
+				<p class="brdr-bttm">
+					Example 4
+				</p>
+				<p class="brdr-0">
+					Example 5
+				</p>
+				<p class="brdr-rds-0">
+					Example 6
```

Comment:
Be more descriptive with the text. Include the expected style. 

```suggestion
					Example of a left border
				</p>
				<p class="brdr-tp">
					Example of a top border
				</p>
				<p class="brdr-rght">
					Example 3
				</p>
				<p class="brdr-bttm">
					Example 4
				</p>
				<p class="brdr-0">
					Example 5
				</p>
				<p class="brdr-rds-0">
					Example 6
```